### PR TITLE
[Plugin] add aiohttp plugin

### PR DESCRIPTION
**Pull request by tom-pytel**
Continuing #100 

- [x] Add a test case for the new plugin
- [ ] Add a component id in [the main repo](https://github.com/apache/skywalking/blob/master/oap-server/server-bootstrap/src/main/resources/component-libraries.yml#L415)
- [x] Add a logo in [the UI repo](https://github.com/apache/skywalking-rocketbot-ui/tree/master/src/views/components/topology/assets)
- [x] Rebuild the `requirements.txt` by running `tools/env/build_requirements_(linux|windows).sh`

### CODE REVIEW

File: skywalking/plugins/sw_aiohttp.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            if val is not None:
+                item.val = val
+
+        with context.new_entry_span(op=request.path, carrier=carrier) as span:
+            span.layer = Layer.Http
+            span.component = Component.Unknown  # TODO: Component.aiohttp
+            span.peer = request.remote
+
+            span.tag(Tag(key=tags.HttpMethod, val=request.method))
+            span.tag(Tag(key=tags.HttpUrl, val=str(request.url)))
```
[the remark this answers is not in the document]
Comment (tom-pytel) [reply skywalking/plugins/sw_aiohttp.py]: :facepalm:

**Comment by kezhenxu94** [comment]
Hi @tom-pytel , I've added the test case for you, some of the fields (`peer`) are not aligned correctly, please adjust according to the expected data, thanks, I'll review the other plugins about the `peer` field that we mentioned in #100

**Comment by tom-pytel** [comment]
> Hi @tom-pytel , I've added the test case for you, some of the fields (`peer`) are not aligned correctly, please adjust according to the expected data, thanks, I'll review the other plugins about the `peer` field that we mentioned in #100

Not just peer, might be possibility of pwd showing up in url field too maybe.

**Review by kezhenxu94 — APPROVED** [review]
The last step is to add the component id to the main repo

https://github.com/apache/skywalking/blob/84af1d23834ef3c6ed17980c5270fc42d6e7d5b3/oap-server/server-bootstrap/src/main/resources/component-libraries.yml#L479

which may only cost 10 seconds for you but I'd appreciate it and the credits should belong to you

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**